### DISCUSSION TITLE
About cabling color and mediatype

### DISCUSSION THREAD

**Discussion by Tarot67** [opendcim/openDCIM#1367] [General]
I version 21.01 and 23.01

For color:

When i try to put a color with API or webui. That doesn't create color.

for adding color by webui in logs log a have nothing paritcular, i try also to create color with put API fonction "colorcode":
[02/Jun/2023:11:57:28 +0200] "PUT /api/v1/colorcode/CCCCCC HTTP/1.1" 200 467 "-" "curl/8.0.1"

But color is not created in database

For MediaType:

But when i try to suppress a MediaType i have this in log:

[php:error] [pid 14] [client 172.19.0.1:56788] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dcim.fac_DevicePorts' doesn't exist in /var/www/html/classes/MediaTypes.class.php:148\nStack trace:\n#0 /var/www/html/classes/MediaTypes.class.php(148): PDOStatement->execute()\n#1 /var/www/html/configuration.php(223): MediaTypes::TimesUsed()\n#2 {main}\n  thrown in /var/www/html/classes/MediaTypes.class.php on line 148, referer: http://127.0.0.1/configuration.php

I see on database and yes table fac_DevicePorts doesn't exsite ?

**Comment by samilliken** [comment]
What version are you on?   Was this an upgrade or a fresh install?   That table has been in the schema for many versions, so that looks…   bad.

 

Scott

 

From: Tarot67 
Sent: Friday, June 02, 2023 6:08 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] About cabling color and mediatype (Discussion #1367)

 

When i try to put a color with API or webui. That doesn't create color i have also a issue when i try to delete a MediaType.

for color in logs log a have nothing paritcular but color is not added:

But when i try to suppres a MediaType i have this in log:

[php:error] [pid 14] [client 172.19.0.1:56788] PHP Fatal error: Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dcim.fac_DevicePorts' doesn't exist in /var/www/html/classes/MediaTypes.class.php:148\nStack trace:\n#0 /var/www/html/classes/MediaTypes.class.php(148): PDOStatement->execute()\n#1 /var/www/html/configuration.php(223): MediaTypes::TimesUsed()\n#2 {main}\n thrown in /var/www/html/classes/MediaTypes.class.php on line 148, referer: http://127.0.0.1/configuration.php

I see on database and yes table ac_DevicePorts doesn't exsite ?

**Comment by Tarot67** [comment]
I restore dump

i have this problem on 21.01, 23.01

I think it's poblalby due to me ;) I made an a error somewhere.








Le ven. 2 juin 2023 à 13:21, Scott Milliken ***@***.***> a
écrit :
